overview/saml.mdx

```diff
@@ -15,6 +15,45 @@ which users have access to your product. They could give everyone at the company
 access, specify teams like only the engineering team, or even pick specific
 people to give access.
 
+### SAML Setup Guide
+
+
+
+1. In the Organization Settings for your project, make sure that SAML is enabled and that all orgs can setup SAML.
+
+![organization settings](/images/saml-setup-1.png)
+
+2. Under Users, make sure that there is at least one test user with Admin or Owner priviledges.
+
+![test user](/images/saml-setup-2.png)
+
+3. On the dashboard for your project, click the "View Org Page" button under the Test Environment.
```

Comment:
If you skip step 2, I'd just add something like.. When you click this, you are now viewing the organization management page for your product. In other words, you are now acting as if you were an employee at Acme Co. If you haven't done it yet, you'll need to sign up before you can view the org page.